lib/m2ts/m2ts.js

```diff
@@ -33,6 +32,9 @@ var
 TransportPacketStream = function() {
   var
     buffer = new Uint8Array(MP2T_PACKET_LENGTH),
+    totalDataEmitted = 0,
+    sinceFlushDataEmitted = 0,
+    invalidPushCalls = 0,
```

Comment:
I think we should remove these until we have a need for them. Also, that means we do not need the layer of indirection between `stream.trigger(data)` so we can go back to using trigger() directly.